src/server/http/HTTPResponder.cpp

```diff
@@ -29,15 +29,45 @@ void HTTPResponder::generateResponse(HTTPClient &client) {
 	try {
 		CGIenv::env envp;
 		envp.setEnv(CGIenv::ENVBuilder()
-				.AUTH_TYPE("authentication")
-				.CONTENT_LENGTH("3000")
-				.CONTENT_TYPE("text/json")
-				.REQUEST_URI("/path")
-				.build()
-				);
+			.AUTH_TYPE(request._headers["AUTHORIZATION"]);
+			.CONTENT_LENGTH(utils::itos(request._body.length()));
```

Comment:
You can change `utils::itos` to `utils::intToString`, no need to make another function